### PR TITLE
v5.0.x: docs updates

### PR DESCRIPTION
**Pull request by jsquyres**
This is the v5.0.x version of #10762.

I'm going to wait until #10771 merges so that I can add it to this PR, and get all those commits in a single PR to v5.0.x.
***EDIT:*** #10771 has been merged, and the commits have been added to this PR.

### CODE REVIEW

**Comment by jsquyres** [comment]
bot:aws:recheck

File: docs/running-apps/tuning.rst [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+          Runtime Environment (ORTE) project.  ORTE essentially
+          evolved into the standalone `PMIx Runtime Reference
+          Environment (PRRTE) <https://github.com/openpmix/prrte>`_
+          and is now considered a 3rd-party dependency of Open MPI
+          |mdash| not one of its included projects.
+
+          See :ref:`the role of PMIx and PRRTE
+          <label-running-role-of-pmix-and-prte>` for more information.
+
+Frameworks
```
[the remark this answers is not in the document]
Comment (jsquyres) [reply docs/running-apps/tuning.rst:67]: You mean more than https://docs.open-mpi.org/en/main/running-apps/pmix-and-prrte.html?

File: docs/running-apps/tuning.rst [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+
+.. note:: Good synonyms for Open MPI's "component" concept are
+          "plugin", or "add-on".
+
+The exact set of components varies between different versions of Open
+MPI.  Open MPI's code base includes support for many components, but
+not all of them may be present or available on your system.  You can
+use the :ref:`ompi_info(1) <man1-ompi_info>` command to see what
+components are included in Open MPI |ompi_ver| on your system.
+
```
[the remark this answers is not in the document]
Comment (jsquyres) [reply docs/running-apps/tuning.rst:109]: Yes, if we mention that PMIx and PRTE have MCA frameworks of their own (e.g., an in Attention box or something), we could also mention `pmix_info` and `prte_info` in another Attention box here, or something).

We'd need to make a new PR to main about this.

[the remark this answers is not in the document]
Comment (rhc54) [reply docs/running-apps/tuning.rst:109]: Yeah, I've been thinking about that - users are going to confront a 3-headed world where they do need to know what the various layers do and how they can be directed. You might need a little overview somewhere so they can visualize it.

**Comment by jsquyres** [comment]
@gpaulsen @rhc54 I think those points are valid; I'll make another PR on main about them.  I think this PR is fine to merge as-is; the docs are an evolving / living document... 😄 ✏️ 👓

File: docs/faq/general-tuning.rst [hunk 3]
```diff
@@ -638,7 +162,8 @@ the ``mpi_warn_on_fork`` MCA parameter.  For example:
    shell$ mpirun --mca mpi_warn_on_fork 0 ...
 
 Of course, systems that ``dlopen("libmpi.so", ...)`` may not use Open
-MPI's ``mpirun``, and therefore may need to use :ref:`a different
+MPI's ``mpirun``, and therefore may need to use (JMS: this ref no
```
[the remark this answers is not in the document]
Comment (jsquyres) [reply docs/faq/general-tuning.rst:165]: This is in the FAQ section of the doc, and is destined to go away -- that's why I didn't bother fixing up the ref here.

**Comment by jsquyres** [comment]
bot:aws:retest

**Comment by jsquyres** [comment]
bot:aws:retest

**Review by awlauria — APPROVED** [review]

**Comment by awlauria** [comment]
> @gpaulsen @rhc54 I think those points are valid; I'll make another PR on main about them. I think this PR is fine to merge as-is; the docs are an evolving / living document... smile pencil2 eyeglasses

Approving and merging based on this.

**Pull request merged**